extensions/actions/ExplorationQuit/ExplorationQuit.py

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Python configuration for recording the action corresponding to a learner
+submitting an answer.
```

Comment:
This is not a "submitting an answer" action -- please don't copy blindly.

Ditto below.

Reply:
Done. Sorry for the error!